client/webserver/site/src/html/markets.tmpl

```diff
@@ -215,22 +208,24 @@
               </div>
 
               {{- /* ORDER FORM */ -}}
-              <form id="orderForm" class="p-3 d-hide">
-                <div class="p-0 w-100">
-
-                  {{- /* BUY - SELL SELECTOR */ -}}
-                  <div class="d-flex justify-content-start pb-2 fs14">
-                    <button id="buyBttn" type="button" class="buygreen-bg selected">
-                      [[[Buy]]]
-                      <span data-unit="base"></span>
-                    </button>
-                    <button id="sellBttn" type="button" class="sellred-bg ms-2">
-                      [[[Sell]]]
-                      <span data-unit="base"></span>
-                    </button>
-                    <span></span>
+              <form id="orderForm" class="p-1 d-hide">
+                {{- /* ORDER TYPE BUTTONS */ -}}
+                <div id="orderTypeBttns" class="fs14 d-flex justify-content-between">
```

Comment:
Nice! Moving these into the form will give us tab navigation as discussed in #2022.